bin/main.rb

```diff
@@ -46,40 +47,78 @@ def ask_move_two
   end
 
   def validate_one
+    compare_one = Play.new
     if @user_one_move.between?(1, 9) && @num_check.none?(@user_one_move)
-      board_game
+      placement_x
+      puts '------------------------------'
+      show_board
+      puts '------------------------------'
       @num_check.push(@user_one_move)
+      @one_acc.push(@user_one_move)
+      compare_one.comparison_one(@one_acc)
     else
-      puts 'You made an invalid move!!'
+      puts '------------------------------'
+      puts '--> You made an invalid move!! <--'
+      puts '------------------------------'
       ask_move_one
     end
   end
 
   def validate_two
+    compare_two = Play.new
     if @user_two_move.between?(1, 9) && @num_check.none?(@user_two_move)
+      placement_o
+      puts '------------------------------'
       board_game
+      puts '------------------------------'
       @num_check.push(@user_two_move)
+      @two_acc.push(@user_two_move)
+      compare_two.comparison_two(@two_acc)
     else
-      puts 'You made an invalid move!!'
+      puts '------------------------------'
+      puts '--> You made an invalid move!! <--'
+      puts '------------------------------'
       ask_move_two
     end
   end
-end
```

Comment:
- [ ] [Please fix]  You are not respecting the DRY rule here, as a developer this rule is very important and it means that you always have to look for ways to not repeat yourself, like creating methods that do exactly the same thing. You need to convert this method to accept a parameter (ex: player) and perform the action depending on the parameter.
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/dry.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/))